VENTokenField/VENToken.h

```diff
@@ -28,6 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (assign, nonatomic) BOOL highlighted;
 @property (copy, nonatomic, nullable) void (^didTapTokenBlock) (void);
+@property (copy, nonatomic, nullable) void (^didCreateTokenBlock) (void);
```

Comment:
is this used anywhere?

Reply:
yep, in the Superhuman codebase to add the drag and drop GR to the view